File: app/src/main/java/com/mdmobile/pocketconsole/gson/devices/WindowsCE.java
package com.mdmobile.pocketconsole.gson.devices;

/**
 * Created by Antonio on 21/05/2017.
 */

public class WindowsCE extends BasicDevice {
    public WindowsCE(String Kind,String deviceId, String deviceName, String enrollmentTime, String family, String
            hostName, String MACAddress, String manufacturer, String mode, String model, String OSVersion, String path, Boolean complianceStatus, Boolean isAgentOnline, Boolean isVirtual) {
        super(Kind,deviceId, deviceName, enrollmentTime, family, hostName, MACAddress, manufacturer, mode, model,
                OSVersion, path, complianceStatus, isAgentOnline, isVirtual);
    }
}

File: app/src/main/java/com/mdmobile/pocketconsole/dataModels/api/devices/WindowsCE.kt
package com.mdmobile.pocketconsole.dataModels.api.devices


class WindowsCE(Kind: String = "N/A", DeviceId: String = "N/A", DeviceName: String = "N/A", EnrollmentTime: String = "N/A",
                Family: String = "N/A", HostName: String = "N/A", MACAddress: String = "N/A", Manufacturer: String = "N/A",
                Mode: String = "N/A", Model: String = "N/A", OSVersion: String = "N/A", Path: String = "N/A",
                ComplianceStatus: Boolean = false, IsAgentOnline: Boolean = false,
                IsVirtual: Boolean = false, Platform: String = "N/A")
    : BasicDevice(Kind, DeviceId, DeviceName, EnrollmentTime, Family, HostName, MACAddress, Manufacturer, Mode,
        Model, OSVersion, Path, ComplianceStatus, IsAgentOnline, IsVirtual, Platform), IDevice<WindowsCE>{
    override fun getDevice(): WindowsCE {
        return this;
    }
}

//TODO : Support windows CE extras